classes/NPRAPIWordpress.php

```diff
@@ -94,10 +94,24 @@ function query_by_url( $url ) {
    * This function will go through the list of stories in the object and check to see if there are updates
    * available from the NPR API if the pubDate on the API is after the pubDate originally stored locally.
    *
+   * @see https://github.com/nprds/nprapi-wordpress/commit/ac0a7f7e3b7428ba783853eb76696b499bc85ecc for is_numeric($args)
+   *
+   * @since 1.7
+   *
    * @param bool $publish
+   * @param array|int $opts { If numeric it will be `'query_number' otherwise an array of named options.
+   *
+   *    @type int $query_number The number of the query for $args['tags_input']
+   *
+   * }
    * @return int|null $post_id or null
    */
-    function update_posts_from_stories( $publish = TRUE ) {
```

Comment:
I will be happy to remove my last changeset if that's what you would prefer.  

But before I do may I ask to make sure you were aware they were based on the suggestions I made to PR #39?  I added comments inline to the changeset but I don't see the comments I added:  https://github.com/nprds/nprapi-wordpress/pull/39/files

I basically suggested that you future proof and not add a numeric 2nd parameter to update_posts_from_stories() but instead add an $opts array that would contain the `query_number` so that in future if you need to add other parameters you would have end up with a list of many ordinal parameters.  

But of course, feel free to decline this modification if you feel it is unimportant in which case I will revert that last commit from my PR.